crates/fluvio-controlplane-metadata/Cargo.toml

```diff
@@ -13,6 +13,7 @@ path = "src/lib.rs"
 
 [features]
 smartmodule = ["flate2","toml","use_serde"]
+smartmodule_validation = ["wasmparser"]
```

Comment:
Is there reason to make this feature flag?

Reply:
I asked for it avoid an increase in the size of the dashboard ui wasm.

Reply:
I'm resolving this given that implementation has changed.